pom.xml

```diff
@@ -242,5 +242,23 @@
                 </dependencies>
             </dependencyManagement>
         </profile>
+
+        <profile>
+            <id>native</id>
+            <activation>
+                <property>
+                    <name>quarkus.profile</name>
+                    <value>native</value>
+                </property>
+            </activation>
+
+            <properties>
+                <quarkus.profile>native</quarkus.profile>
```

Comment:
Remove this line, to avoid impression that it might have an effect.

Reply:
@mjurc you forgot about this?